### PR TITLE
Chapter6

### PR DESCRIPTION
**Pull request by NurAnisAqilah**
ready

### CODE REVIEW

**Comment by NurAnisAqilah** [comment]
ready

File: app/models/user.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    has_secure_password
+  
+    before_save :downcase_email
+  
+    private
+  
+    def downcase_email
+      email.downcase!
+    end
+  end
```
Comment (saobv-2223) [inline app/models/user.rb]: indenting only 2 spaces

**Comment by NurAnisAqilah** [comment]
ready

**Review by saobv-2223 — COMMENTED** [review]
merge

**Pull request merged**